bulletin/index.html

```diff
@@ -34,137 +34,136 @@ <h1 class="logo-brand" itemprop="brand">MACILE</h1>
+			</article>
+			<article class="buarticle">
+				<h3 class="buarticle-main--title">Schedule and evaluation</h3>
+				<p>
+				 The school day was extended from 12:00 PM to 4:00 PM and students were provided with a healthy lunch as well as breakfast. The student's mathematical and linguistic progress was gauged through a math test and an evaluation questionnaire completed at the end of the program.
+				</p>
+			</article>
+			<article class="buarticle">
+				<img src="/images/bulletin/SA2007/itabo-2007-s.jpg">
+				<p>MACILE Summer Academy 2007 and 2008 were conduceted in July at the Itabo Elementary School in Itabo, San Critóbal, Dominican Republic.</p>
+			</article>
+			<article class="buarticle">
+				<img src="/images/bulletin/espanol-2008-4.jpg">
+				<p>Spanish teacher, Antonia Asencio, and members of her class, produly displaying resources use to improve their learning.</p>
+			</article>
+			<article class="buarticle">
+				<h3 class="buarticle-main--title">Los indios de Quisqueya</h3>
+				<h4 class="buarticle-subheading">MACILE Spanish currilum</h4>
+				<p>An innovative active approach to deepen understanding of the language through the history and folblore of the Dominican Republic. A book written exclusively for MACILE by Dr. Rosario Montelongo de Swanson.</p>
+				<p>Los indios de Quisqueya, the accompaning history book and foundation of the MACILE curriculum, was donated by the Tavares Foundation. The author, Juan Tómas Tavares Kelnes, permited its use.</p>
+			</article>
+			<article class="buarticle">
+				<img src="/images/bulletin/sa2008-spanish-s.jpg">
+				<p>Learning language and enjoying the folklore. Students dancing at the bits of the grum and güira, singing the <i>salve</i>,learning Spanish.</p>
+			</article>
+			<article class="buarticle">
+				<img src="/images/bulletin/eng-tech-sa2008-s.jpg">
+				<p>Students built toothpick bridges and learned about solar energy. This team is testing their bridge resistance.</p>
+			</article>
+			<article class="buarticle">
+				<img src="/images/bulletin/group-2008-014.jpg">
+				<p>MACILE <i>nourishes the innate curiosity and wonderful imagination of young people rather than discourage them</i>, as Einstein admonished, to expand their horizons.</p>
+			</article>
+			<article class="buarticle">
+				<img src="/images/bulletin/busLeaders-2008-m.jpg">
+				<p>Visit from Dominican business leader and gov officials. They spoke about the job markets and the government scholarship program. Students learned about opportunities and requirements to be successful.</p>
+			</article>
+			<article class="buarticle">
+				<img src="/images/bulletin/tdi-2008-math_3.jpg">
+				<h4 class="buarticle-heading">Teacher Development</h4>
+				<p>Dominican math teachers participating in two weeks long math training workshop: improving learning of basic algebra.</p>
```

Comment:
* participating in a two week long math training workshop: Improving Learning of Basic Algebra